src/pages/home/report/ReportActionItemParentAction.tsx

```diff
@@ -0,0 +1,92 @@
+import React from 'react';
+import {View} from 'react-native';
+import {withOnyx} from 'react-native-onyx';
+import type {OnyxEntry} from 'react-native-onyx';
+import OfflineWithFeedback from '@components/OfflineWithFeedback';
+import type {WithLocalizeProps} from '@components/withLocalize';
+import type {WindowDimensionsProps} from '@components/withWindowDimensions/types';
+import useStyleUtils from '@hooks/useStyleUtils';
+import useThemeStyles from '@hooks/useThemeStyles';
+import * as ReportActionsUtils from '@libs/ReportActionsUtils';
+import * as Report from '@userActions/Report';
+import ONYXKEYS from '@src/ONYXKEYS';
+import type * as OnyxTypes from '@src/types/onyx';
+import AnimatedEmptyStateBackground from './AnimatedEmptyStateBackground';
+import ReportActionItem from './ReportActionItem';
+
+type ReportActionItemParentActionOnyxProps = {
+    /** ONYX PROPS */
+
+    /** The report currently being looked at */
+    report: OnyxEntry<OnyxTypes.Report>;
+
+    /** The actions from the parent report */
+    // TO DO: Replace with HOC https://github.com/Expensify/App/issues/18769.
+    parentReportActions: OnyxEntry<OnyxTypes.ReportActions>;
+};
+type ReportActionItemParentActionProps = WithLocalizeProps &
+    WindowDimensionsProps &
+    ReportActionItemParentActionOnyxProps & {
+        /** Flag to show, hide the thread divider line */
+        shouldHideThreadDividerLine: boolean;
+
+        /** Flag to display the new marker on top of the comment */
+        shouldDisplayNewMarker: boolean;
+
+        /** The id of the report */
+        reportID: string;
+
+        /** The id of the parent report */
+        // eslint-disable-next-line react/no-unused-prop-types
+        parentReportID: string;
+    };
+
+function ReportActionItemParentAction({report, parentReportActions, isSmallScreenWidth, shouldHideThreadDividerLine, shouldDisplayNewMarker}: ReportActionItemParentActionProps) {
```

Comment:
@sourcecodedeveloper Can you please add the default props that were there before here?

Reply:
As an example, the default prop for report should be defined like:
```
function ReportActionItemParentAction({report = {}, parentReportActions, isSmallScreenWidth, shouldHideThreadDividerLine, shouldDisplayNewMarker}: ReportActionItemParentActionProps) {
```

Reply:
report = {} gives error,  Property 'reportID' is missing in type '{}' but required in type 'Report'.

I added default props for
    parentReportActions = {},
    shouldHideThreadDividerLine = false,

Reply:
But why does it give that error? In the previous implementation, we were using the same `{}` as the default prop.

Reply:
I resolved the error, reportID also needed to be set with default value